src/frontends/tensorflow_common/include/utils.hpp

```diff
@@ -34,13 +34,26 @@ void set_node_name(const std::string& node_name, const std::shared_ptr<Node>& no
 bool is_conditional_edge(const std::string& input_tensor_name);
 
 template <typename T>
-ov::Output<ov::Node> create_same_type_const_scalar(const ov::Output<ov::Node>& same_type_output, T value) {
+ov::Output<ov::Node> create_same_type_const_scalar(const ov::Output<ov::Node>& same_type_output, const T& value) {
```

Comment:
Both of these functions are looks like as some workaround. Would be great to see a documentation for them. And especially cases when they should be used.

Reply:
They are not workarounds and they are more like helpers to avoid replicate code to avoid dynamic types.
Let me elaborate here - what this helper is doing. In cases of FW nodes appeared for unsupported operations in the graph, we can face dynamic types for outputs and usage of `get_element_type()` in translators can stop the conversion. In order to avoid it. we can compute the type or align it in the run-time using `ConvertLike`.